src/TurbulenceConvection/update_aux.jl

```diff
@@ -53,9 +54,6 @@ function update_aux!(
     ##### center variables
     #####
     C123 = CCG.Covariant123Vector
-    @. aux_en.e_kin =
```

Comment:
this is wrong, w of env is not updated here yet

Reply:
Oh no! Is this an issue in TC, too?

Reply:
I undid this change as it has some weird effects I need to investigate separately. lets merge just adding gm_w without behavior changes first ?